### PR TITLE
Jetty 10 with JSR 356 websocket working

### PR DESCRIPTION
**Pull request by ddworak**

### CODE REVIEW

File: rest/src/test/scala/io/udash/rest/RestApiTest.scala [hunk 1]
```diff
@@ -67,7 +67,7 @@ trait RestApiTestScenarios extends RestApiTest {
   }
 
   test("prefixed GET") {
-    testCall(_.prefix("", "h0", "q0").subget(0, 1, 2))
+    testCall(_.prefix("p0", "h0", "q0").subget(0, 1, 2))
```
Comment (sebaciv) [inline rest/src/test/scala/io/udash/rest/RestApiTest.scala:70]: Empty prefix no longer works here?

Comment (ddworak) [reply rest/src/test/scala/io/udash/rest/RestApiTest.scala:70]: Default compliance settings do not allow double slash in URL, as defined in the RFC. They can be overriden individually, but I did not think it important for tests here - do you?

Comment (sebaciv) [reply rest/src/test/scala/io/udash/rest/RestApiTest.scala:70]: Does `@Prefix("")` usages break in any way? If not - it's OK

Comment (ddworak) [reply rest/src/test/scala/io/udash/rest/RestApiTest.scala:70]: `io.udash.rest.RestTestApi#transparentPrefix` is not broken. Empty prefix does not mean an empty URL part (double slash), right?

**Pull request merged**